src/egon/data/processing/heat_demand_timeseries/nextcloud link

```diff
@@ -0,0 +1,3 @@
+Data for running the code are availabe in the following link:
+				https://next.rl-institut.de/s/xSKCwBtqzdZ4spd
```

Comment:
The hdf5-file will be added to the databundle of egon-data on zenodo soon. I would like to merge this afterwards to avoid this manual download. Do you need any other file(s) as an input for you code?

Reply:
Other than the hdf-file, the shape files of the climate zones is also needed which is available in the same link in the folder TRY_Climate_Zones

Reply:
The shape files for the climate zones are already part of the data bundle: `data_bundle_egon_data/climate_zones_germany/TRY_Climate_Zone`
As soon as the profiles are also included I will write you the path and name of the file(s).